src/components/AvatarHeader/AvatarHeader.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { useParams, useHistory } from 'react-router';
+import { Box, createStyles, Theme } from '@material-ui/core';
+import makeStyles from '@material-ui/core/styles/makeStyles';
+import CircularProgress from '@material-ui/core/CircularProgress';
+import { useQuery } from '@apollo/react-hooks';
+import Avatar from 'components/Avatar';
+import AVATAR_HEADER from '../../graphql/avatar-header.graphql';
+
+import { Query, QueryNewbieArgs } from '../../types';
+import { getProgressInPercentages } from '../../utils';
+import { ROUTES } from '../../shared/routes';
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    background: {
+      background: theme.palette.background.paper,
+      display: 'flex',
+      justifyContent: 'center',
+      alignItems: 'center',
+      width: '100%',
+      minHeight: '30rem',
+    },
+  })
+);
+
+const AvatarHeader: React.FC = () => {
+  const history = useHistory();
+  const { background } = useStyles();
+  const { newbieId } = useParams<QueryNewbieArgs>();
+
+  const { loading, data } = useQuery<Query, QueryNewbieArgs>(AVATAR_HEADER, {
```

Comment:
Shouldn't be only a visual stateless component? I was seeing this component to receive only the avatar information and the parent component (taskList) does the query to fetch the tasks plus the newbie avatar information and just passes the data to this one.

Reply:
In my opinion, @lucianojss is right. [BUD-26](https://digitalrig.atlassian.net/browse/BUD-26) is the ticket for that - AvatarHeader.tsx - should be pure component and in BUD-26 we should create a wrapper for the whole Task List page, group all elements and add graphQL data distribution only there.

Reply:
It could be but since this component will be used in different views I've decided to make it autonomic. This component will be used in the different views (screenshot) and we don't know how we gonna implement them.... I'm gonna leave it as it is and we can think of improvements when all those view will be ready. :point_down:
![image](https://user-images.githubusercontent.com/14184100/68961519-e1244a80-07d2-11ea-8ea2-da08bb3d5a04.png)